src/sdk-reference/cpp/1/websocket/unregister-sub/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: sdk.html.hbs
+title: unregisterSub
```

Comment:
Please apply to this page all the suggestions I made for the same method documentation in the Protocol class